test/Microsoft.Azure.Devices.Edge.Test/Module.cs

```diff
@@ -12,6 +12,35 @@ namespace Microsoft.Azure.Devices.Edge.Test
 
     public class Module : ManualProvisioningFixture
     {
+        private sealed class TempSensorModule
+        {
+            public string Name { get; }
+            public string Image { get; }
+
+            private const string DefaultSensorImage = "mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:1.0";
+            // Make sure it's thread-safe
+            private static readonly object padlock = new object();
+            private static int instanceCount = 0;
+
+            private TempSensorModule()
```

Comment:
Did StyleCop warn about this? I'd remove it since it's empty, it's never used, and using it would put the object in a bad state (`Name` and `Image` would be `null` I think).